river/preprocessing/ordinal.py

```diff
@@ -0,0 +1,91 @@
+import collections
+import math
+import numbers
+
+import numpy as np
+import pandas as pd
+
+from river import base
+
+
+class OrdinalEncoder(base.MiniBatchTransformer):
+    # INFO: makes sense to add None initially? so it'll be easier to spot
+    def __init__(self, handle_unknown="use_reserved_value", unknown_value=1, encoded_missing_value=0):
+        # self.categories = categories
+        # self.dtype = dtype
+        self.handle_unknown = handle_unknown
+        self.unknown_value = unknown_value
+        self.encoded_missing_value = encoded_missing_value
+        self.categories_ = collections.defaultdict(dict)
+        # self.categories_ = collections.defaultdict(collections.defaultdict)
+
+    def learn_one(self, x):
+        for i, xi in x.items():
+            if self.handle_unknown == "use_reserved_value":
```

Comment:
So what happens in the other case? Nothing?

Reply:
Open to suggestions! 

At the moment i just implemented what was relevant in my usecase.

Mimicking `sklearn` api here a bit. Tho raising an error for unkowns like `sklearn` does would be weird in online setting. Unless there's an option to supply a predefined mapping values -> encodings. Then unknown values start make sense in online setting. 
But i'm not sure we want to support that in `OrdinalEncoder` - user can just use `FunctionTransformer` if they want a mapping.

The only other option is to have the unknowns and the missing values map to the same code, as mentioned in the comments above. I can implement that.